src/popup/router/components/Modals/RecipientInfo.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <Default
+    v-bind="{ ...$attrs, resolve }"
+    icon="help"
+    :title="$t('modals.recipient.title')"
+    :close="resolve"
+  >
+    <template #msg>
+      <div
+        class="msg"
+      >
+        <span class="sub-header">
+          {{ $t('modals.recipient.sub-header') }}
+        </span>
+        <p>
+          <i18n
+            path="modals.recipient.msg.publicAddress.msg"
+            tag="div"
+          >
+            <strong class="title">
+              {{ $t('modals.recipient.msg.publicAddress.title') }}:
+            </strong>
+          </i18n>
+        </p>
+        <p>
+          <i18n
+            path="modals.recipient.msg.chain.msg"
+            tag="div"
+          >
+            <strong class="title">
+              {{ $t('modals.recipient.msg.chain.title') }}:
+            </strong>
+            <RouterLink :to="{ name: 'name-claim' }">
+              {{ $t('modals.recipient.msg.chain.linkTitle') }}
+            </RouterLink>
```

Comment:
As discussed in the call, this is no more a link.
@CedrikNikita while cherry-picking, could you please change this from `RouterLink` to `Span`